src/components/CloudHeader/_cloud-header.scss

```diff
@@ -0,0 +1,119 @@
+@import 'carbon-components/scss/globals/scss/colors';
+@import 'carbon-components/scss/globals/scss/helper-mixins';
+
+$cloud-header-link-color: $inverse-01 !default;
+$cloud-header-icon-color: $inverse-01 !default;
+$cloud-header-menu-color: $brand-02 !default;
+$cloud-header-background-color: $color__blue-90 !default;
+
+.cloud-header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  background-color: $cloud-header-background-color;
+  color: $cloud-header-link-color;
+  height: rem(48px);
+  width: 100%;
+
+  a {
+    color: $cloud-header-link-color;
+    text-decoration: none;
+    transition: color $transition--base $bx--standard-easing;
+
+    &:hover,
```

Comment:
Style nit: nested selectors.

Reply:
`hover` and `focus` are fine to nest as long as it is understandable, which in this case I think it is.